### DISCUSSION TITLE
how to fix The problem doesn't have a solution with actual input parameters

### DISCUSSION THREAD

**Discussion by BlockWaving** [dcajasn/Riskfolio-Lib#20] [Q&A]
Hi dcajasn,
seached for cvxpy and find this new wrapper maybe a better start for me.  
Just posted a question on the Issue column, but the kind of Q&A may be better placed here:

I’m trying to adapt couple of samples to test on a custom alpha estimates, using an industry constraints and MaxRet target, but the optimizer consistently gives out this:

The problem doesn't have a solution with actual input parameters

Here is the constrains:

Disabled | ---Type-- | ----Set ----| Position | Sign | Weight | Type Relative | Relative Set | Relative | Factor
FALSE ---| All Assets |------------|---------- | <= | 0.08 | | | |
FALSE ---| All Classes | Industry |----------- | <= | 0.25 | | | |
FALSE--- | All Classes | Industry |----------- | >= | 0.03 | | | |

There are around 250 assets (stocks) and their respective Industries have been mapped, and the asset_classes matrix has been created.

What may have been the reason that the ‘solution’ couldn’t be found? Where should I look at to find the fix?

Thanks,

**Comment by JeretQ** [comment]
maybe you didn't put the mosek lisence in the right folder
